### PR TITLE
Add the new plot of the M_start - M_BH relation split by active and passive

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
Needs https://github.com/SWIFTSIM/velociraptor-comparison-data/pull/145

### CODE REVIEW

File: colibre/auto_plotter/black_holes.yml [hunk 1]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
     - filename: BlackHoleMassFunction/Shankar2009_Data.hdf5
     - filename: BlackHoleMassFunction/Vika2009_Data.hdf5
     - filename: BlackHoleMassFunction/Caramete2010_Data.hdf5
+
+stellar_mass_black_hole_mass_30:
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:320]: We already have a plot with the key name `stellar_mass_black_hole_mass_30` so we can't use it again.
```suggestion
stellar_mass_black_hole_mass_passive_30:
```

File: colibre/auto_plotter/black_holes.yml [hunk 2]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (30 kpc Stellar Mass) for passive galaxies.
+    caption: SMBHM relation for the 30 kpc stellar mass aperture. 
+    section: Black Holes
+    show_on_webpage: false
+  observational_data:
+    - filename: GalaxyStellarMassBlackHoleMass/Terrazas2017_NSF.hdf5
+
+stellar_mass_black_hole_mass_50:
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:355]:
```suggestion
stellar_mass_black_hole_mass_passive_50:
```

File: colibre/auto_plotter/black_holes.yml [hunk 3]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (30 kpc Stellar Mass) for active galaxies.
+    caption: SMBHM relation for the 30 kpc stellar mass aperture. 
+    section: Black Holes
+    show_on_webpage: false
+  observational_data:
+    - filename: GalaxyStellarMassBlackHoleMass/Terrazas2017_SF.hdf5
+
+stellar_mass_black_hole_mass_50:
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:424]:
```suggestion
stellar_mass_black_hole_mass_active_50:
```

File: colibre/auto_plotter/black_holes.yml [hunk 4]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (50 kpc Stellar Mass) for passive galaxies.
+    caption: SMBHM relation for the 50 kpc stellar mass aperture. 
+    section: Black Holes
+  observational_data:
+    - filename: GalaxyStellarMassBlackHoleMass/Terrazas2017_NSF.hdf5
+
+stellar_mass_black_hole_mass_30:
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:389]:
```suggestion
stellar_mass_black_hole_mass_active_30:
```

File: colibre/auto_plotter/black_holes.yml [hunk 5]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (30 kpc Stellar Mass) for passive galaxies.
+    caption: SMBHM relation for the 30 kpc stellar mass aperture.
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:349]:
```suggestion
    caption: Stellar Mass-Black Hole Mass relation for the 30 kpc stellar mass aperture. Only passive galaxies (threshold is 0.01 / Gyr sSFR) are included.
```

File: colibre/auto_plotter/black_holes.yml [hunk 6]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (50 kpc Stellar Mass) for passive galaxies.
+    caption: SMBHM relation for the 50 kpc stellar mass aperture.
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:384]:
```suggestion
    caption: Stellar Mass-Black Hole Mass relation for the 50 kpc stellar mass aperture. Only passive galaxies (threshold is 0.01 / Gyr sSFR) are included.
```

File: colibre/auto_plotter/black_holes.yml [hunk 7]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (30 kpc Stellar Mass) for active galaxies.
+    caption: SMBHM relation for the 30 kpc stellar mass aperture.
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:418]:
```suggestion
    caption: Stellar Mass-Black Hole Mass relation for the 30 kpc stellar mass aperture. Only active galaxies (threshold is 0.01 / Gyr sSFR) are included.
```

File: colibre/auto_plotter/black_holes.yml [hunk 8]
```diff
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
...
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Black Hole Mass relation (50 kpc Stellar Mass) for active galaxies.
+    caption: SMBHM relation for the 50 kpc stellar mass aperture.
```
Comment (EvgeniiChaikin) [inline colibre/auto_plotter/black_holes.yml:453]:
```suggestion
    caption: Stellar Mass-Black Hole Mass relation for the 50 kpc stellar mass aperture. Only active galaxies (threshold is 0.01 / Gyr sSFR) are included.
```

**Comment by MatthieuSchaller** [comment]
All good suggestions. Done!

**Comment by EvgeniiChaikin** [comment]
All looks good @MatthieuSchaller! The last remaining is to update the version of the submodule. Its latest version already has your changes merged.

**Review by EvgeniiChaikin — APPROVED** [review]

**Comment by MatthieuSchaller** [comment]
Done.

**Pull request merged**